crates/secret-service-server/src/ms2sm.rs

```diff
@@ -0,0 +1,252 @@
+use std::{mem::MaybeUninit, sync::Arc};
```

Comment:
What is this palindrome `ms2sm`? Can't we have a more intuitive naming or module-level docs?